kernel/memory/src/paging/mod.rs

```diff
@@ -104,6 +104,10 @@ impl PageTable {
                 frame.as_allocated_frame(),
                 PteFlagsArch::new().valid(true).writable(true),
             );
+            table[INACTIVE_PAGE_TABLE_RECURSIVE_P4_INDEX].set_entry(
+                frame.as_allocated_frame(),
+                PteFlagsArch::new().valid(true).writable(true),
+            );
```

Comment:
i realize doing this here is a bit faster, but i wonder if we should leave this P4 entry zeroed-out (invalid) just to ensure that no one can access these addresses at any point (except for, of course, in `PageTable:;with()`.

thoughts?